### Paramteres used by the controller node ###
controller_node:
  ros__parameters:
    #----------------------------------------------------------------------------------------------------
    # AUV
    mass: 1862.87
    volume: 1.83826
    r_cog: [0.0, 0.0, 0.0]
    r_cob: [0.0, 0.0, -0.3]
    Ma: [780.0, 1222.0, 3660.0, 535.0, 842.0, 224.0]
    Dlinear:  [-75.0, -70.0, -728.0, -268.0, -309.0, -105.0]
    Dquad: [-748.2, -992.0, -1821.0, -672.0, -774.0, -523.0]
    Ib: [525.39, 794.2, 691.23, 1.44, 33.41, 2.6]
    #----------------------------------------------------------------------------------------------------
    # Controller Params
    T: 4
    dt: 0.05
    Q: [1e4, 1e4, 1e4, 1e2, 1e2, 1e4, 1e3, 1e3, 1e3, 1e2, 1e2, 1e3]
    R1: [1e0, 1e0, 1e0, 1e0, 1e0, 1e0]
    R2: [0e1, 0e1, 0e1, 0e1, 0e1, 0e1]
    error_max:  [100.0, 100.0, 100.0, 3.14, 3.14, 3.14, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
    tau_max: [1000.0, 1000.0, 1000.0, 1000.0, 1000.0, 1000.0]
    #----------------------------------------------------------------------------------------------------
    # Guidance
    # translation_[velocity, acceleration, jerk]_limits
    translation_constraints: [0.6, 0.35, 0.0005]
    # rotation_[velocity, acceleration, jerk]_limits
    rotation_constraints: [0.4, 0.22, 0.0005]
    # Radius of accepetance for waypoints switching
    radius_of_accepetance: 0.25
    
    
    

    
